services/QuillLMS/app/models/school.rb

```diff
@@ -209,4 +209,10 @@ def subscription_status
       .where(user: old_district_admins)
       .destroy_all
   end
+
+  private def zipcode_length
```

Comment:
Two thoughts:
1) we may want to allow for 9-digit zip codes, or for non-US style zipcodes.  Maybe just make this a minimum rather than a strict length requirement?
2) We can use a Rails helper rather than writing a validation function.  This can be done with: `validates :zipcode, length: {min: 5}, allow_blank: true`